.github/scripts/tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/env bash
+set -eu
+
+mkdir .osc
```

Comment:
Are you sure where you are running this command?
you may want to explicit path like `mkdir ~/.osc`